Core/Core/Common/CommonModels/AppEnvironment/AppEnvironmentOverride.swift

```diff
@@ -109,19 +125,49 @@ public final class AppEnvironmentOverride: AppEnvironment {
         }
         set {}
     }
+
+    public override func transformContentIDs(params: [String: String], url: URLComponents) -> ([String: String], URLComponents) {
```

Comment:
I would extend the name to tell the reader that what IDs are transformed to. `transformContetIDsToLocalIDs` maybe?